tests/standalone/cursor-getmore-001.phpt

```diff
@@ -0,0 +1,36 @@
+--TEST--
+MongoDB\Driver\Cursor query result iteration with batchSize requiring getmore
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; CLEANUP(STANDALONE) ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(STANDALONE);
+
+$bulkWrite = new MongoDB\Driver\BulkWrite;
+
+for ($i = 0; $i < 5; $i++) {
+    $bulkWrite->insert(array('_id' => $i));
+}
+
+$writeResult = $manager->executeBulkWrite(NS, $bulkWrite);
+printf("Inserted: %d\n", $writeResult->getInsertedCount());
+
+$cursor = $manager->executeQuery(NS, new MongoDB\Driver\Query(array(), array('batchSize' => 2)));
+
+foreach ($cursor as $document) {
+    printf("Found ID: %d\n", $document['_id']);
+}
+
+?>
+===DONE===
+<?php exit(0); ?>
+--EXPECT--
+Inserted: 5
+Found ID: 0
+Found ID: 1
+Found ID: 2
+Found ID: 3
+Found ID: 4
+===DONE===
```

Comment:
Since you are adding tests for this -- maybe add one test where the results fit nicely within the batchSize?